storage/innobase/dict/dict0load.cc

```diff
@@ -3224,3 +3224,55 @@ dict_load_foreigns(
 	ut_free(pcur.old_rec_buf);
 	DBUG_RETURN(DB_SUCCESS);
 }
+
+/** Get the system tablespace index root page
+@param root_pages list of index root pages
+@return error code */
+dberr_t dict_get_sys_tablespace_indexes(
+  std::vector<uint32_t> *root_pages)
+{
+  /* Add system table index root pages */
+  dict_sys.sys_tables->add_index_root_pages(root_pages);
+  dict_sys.sys_indexes->add_index_root_pages(root_pages);
+  dict_sys.sys_columns->add_index_root_pages(root_pages);
+  dict_sys.sys_fields->add_index_root_pages(root_pages);
+  dict_sys.sys_foreign->add_index_root_pages(root_pages);
+  dict_sys.sys_foreign_cols->add_index_root_pages(root_pages);
+  dict_sys.sys_virtual->add_index_root_pages(root_pages);
```

Comment:
These should be added by the `dict_getnext_system_low()` loop below. Is that not the case?